### PR TITLE
Task dev 1

### PR DESCRIPTION
**Pull request by ZhylinskayaKseniya**

### CODE REVIEW

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]
deleted

File: README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-for test automation coursers
```
Comment (AlexanderSoroka) [inline README.md:1]: This change doesn't relate to the task - it must not be in that pull-request

File: Task about number from 0 to 100 [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class Dev {
```
Comment (AlexanderSoroka) [inline Task about number from 0 to 100:1]: Inappropriate class name

File: Task about number from 0 to 100 [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  int dev[] = new int[101];
```
Comment (AlexanderSoroka) [inline Task about number from 0 to 100:2]: inappropriate array name

Comment (AlexanderSoroka) [reply Task about number from 0 to 100:2]: You don't actually need array for that task

**Comment by ZhylinskayaKseniya** [comment]
What you mean?
What exactly is wrong?

File: Task about number from 0 to 100 [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  public void enter() {
```
Comment (AlexanderSoroka) [inline Task about number from 0 to 100:3]: inappropriate method name

File: Task about number from 0 to 100 [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    for (int i = 0; i <= 100; i++) {
+      dev[i] = i;
+    }
+  }
+  public void show() {
+    for (int i = 0; i <= 100; i++) {
+      if(dev[i] % 3 == 0) {
```
Comment (AlexanderSoroka) [inline Task about number from 0 to 100:10]: codestyle violation

File: Task about number from 0 to 100 [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        System.out.print("3*" + dev[i]/3 + "\n");
+      }
+        if(dev[i] % 3 != 0) {
```
Comment (AlexanderSoroka) [inline Task about number from 0 to 100:13]: codestyle violation

File: Task about number from 0 to 100 (hunk 5 above)
Comment (AlexanderSoroka) [inline Task about number from 0 to 100:10]: Use if/else instead double if

File: Task about number from 0 to 100 [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	  System.out.print(dev[i] + " ");
+	}
+     }
+  }
+  public static void main(String[] args) {
+    Dev d = new Dev();
```
Comment (AlexanderSoroka) [inline Task about number from 0 to 100:19]: inappropriate one letter name

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Pull request closed without merging**